RecoParticleFlow/PFClusterProducerAlpaka/plugins/AlpakaPFCommon.h

```diff
@@ -0,0 +1,91 @@
+#ifndef RecoParticleFlow_PFClusterProducerAlpaka_plugins_AlpakaPFCommon_h
+#define RecoParticleFlow_PFClusterProducerAlpaka_plugins_AlpakaPFCommon_h
+
+#include <cmath>
+
+namespace PFClustering {
+  namespace common {
+    enum PFLayer {
```

Comment:
Would `enum class` work? Or are the numerical values used more often than not?

Reply:
I realized this file was an unnecessary inclusion from earlier development stages, I removed it and instead use the Layer definitions in `DataFormats/ParticleFlowReco/interface/PFLayer.h`